ctfdpy/ctfdpy/http.py

```diff
@@ -1,3 +1,77 @@
+from enum import Enum
+from json import JSONDecodeError
+from typing import Any
+
+import httpx
+
+
+class CTFDError(Exception):
+    """Exception class to handle exceptions thrown for any request regarding CTFd API"""
```

Comment:
Will certainly need to be put in a separate file in the future to handle more exceptions for particular cases and better UX